packages/cli/src/lib/implementation/only-plugin-options.integration.test.ts

```diff
@@ -0,0 +1,35 @@
+import { describe, expect } from 'vitest';
+import { objectToCliArgs } from '@code-pushup/utils';
+import { yargsCli } from '../yargs-cli';
+import { yargsOnlyPluginsOptionsDefinition } from './only-plugins-options';
+
+describe('onlyPlugin option', () => {
+  type OnlyPluginsOptions = { onlyPlugins?: string | string[] };
+  function argsFromCli<T extends OnlyPluginsOptions>(
+    args: Record<string, unknown>,
+  ) {
+    return yargsCli<T>(objectToCliArgs(args), {
+      options: yargsOnlyPluginsOptionsDefinition(),
+    }).parseAsync() as unknown as Promise<OnlyPluginsOptions>;
+  }
+
+  it('should fill defaults', async () => {
+    const args = await argsFromCli({});
+    expect(args).toBeDefined();
+    // "_" and "$0" are in by default
+    expect(Object.keys(args)).toHaveLength(4);
```

Comment:
Why 4? Based on the comment, I would expect 2.

Reply:
added comments